deployer/templates/es.yaml

```diff
@@ -41,6 +41,7 @@ objects:
             component: ${ES_CLUSTER_NAME}
             deployment: ${ES_DEPLOYMENT_NAME}
         spec:
+          terminationGracePeriodSeconds: 1800
```

Comment:
I think half an hour may be a little over the top... If it's taking that kind of time it's probably hung anyway. What do you think about 10 minutes?